src/infra/marketplace/marketplaceCmdRepo.go

```diff
@@ -437,6 +444,43 @@ func (repo *MarketplaceCmdRepo) InstallItem(
 	)
 }
 
+func (repo *MarketplaceCmdRepo) parseUninstallCmdSteps(
+	filesToDelete []valueObject.UnixFileName,
+	installedSlug valueObject.MarketplaceItemSlug,
+) ([]valueObject.MarketplaceItemCmdStep, error) {
+	uninstallCmdSteps := []valueObject.MarketplaceItemCmdStep{}
+
+	trashDirName := installedSlug.String() + "-%installHostname%-%installUuid%"
+
+	createTrashDirToMoveFilesStep, err := valueObject.NewMarketplaceItemCmdStep(
+		"mkdir -p " + trashDirName,
+	)
+	if err != nil {
+		return uninstallCmdSteps, err
+	}
+
+	firstFileToDelete := filesToDelete[0]
+	filesToDeleteAsParams := "-name \"" + firstFileToDelete.String() + "\""
+
+	filesToDeleteWithoutFirstOne := filesToDelete[1:]
+	for _, fileToDelete := range filesToDeleteWithoutFirstOne {
+		filesToDeleteAsParams += " -o -name \"" + fileToDelete.String() + "\""
+	}
+
+	moveFilesToTrashCmdStepStr := "find %installDirectory% \\( " +
+		filesToDeleteAsParams + " \\) -maxdepth 1 -exec mv -t" + trashDirName + " {} +"
+	moveFilesToTrashCmdStep, err := valueObject.NewMarketplaceItemCmdStep(moveFilesToTrashCmdStepStr)
+	if err != nil {
+		return uninstallCmdSteps, err
+	}
```

Comment:
Did you try using template here? Anyway, I wouldn't set it as a CmdStep, I would just go ahead and delete the files.

Reply:
I already tried, but it becomes overly complex, much more so than simply using concatenation. So, I preferred to keep it as a command.